.eslintrc.yaml

```diff
@@ -82,6 +82,8 @@
           rules:
             import/no-unresolved: off
             default-case: off
+            consistent-return: off
+            react-hooks/exhaustive-deps: off
```

Comment:
You should leave these enabled.

Reply:
Because of our logic, we're running into linter errors.

Reply:
Which logic specifically is causing this problem?